flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -493,7 +537,21 @@ def get_comp_idx(self, comp_dict: dict, error_info: str = "no information") -> i
     def get_ncomp(self) -> int:
         return len(self.compartments)
 
-    def get_transition_array(self):
+    def get_transition_array(self) -> tuple:
```

Comment:
`tuple` is fine for now if it would be onerous to determine the types of its components, but sounds like from the return description they might be known?